.travis.yml

```diff
@@ -14,4 +14,4 @@ install:
  - cd $HOME/gopath/src/k8s.io/node-problem-detector
 script:
  - make test
- - go build -race
+ - make node-problem-detector 
```

Comment:
In #39, we changed this to also build container, but in this PR, it's fine to only binary for now.